lib/events.js

```diff
@@ -525,3 +525,26 @@ function unwrapListeners(arr) {
   }
   return ret;
 }
+
+EventEmitter.prototype.onMultiple = function onMultiple(obj) {
+
+  // Type checking
+  if (typeof obj == 'object') {
+
+    if (Array.isArray(obj)) {
+      throw new Error('Input cannot be an array');
```

Comment:
Any new errors should make use of the internal/errors.
This would use the code `ERR_INVALID_ARG_TYPE`

Reply:
Ops! My bad. I'll correct it as soon as possible. Thanks :+1: